packages/rs-drive-abci/src/execution/storage/fetch_execution_state/mod.rs

```diff
@@ -0,0 +1,29 @@
+use crate::error::execution::ExecutionError;
+use crate::error::Error;
+use crate::platform_types::platform_state::PlatformState;
+use dpp::version::PlatformVersion;
+use drive::drive::Drive;
+use drive::query::TransactionArg;
+
+mod v0;
+
+/// Fetches execution state from grovedb storage
+pub fn fetch_execution_state(
```

Comment:
This should be in Drive instead based on it's parameters

Reply:
Drive doesn't know about execution state

Reply:
Yes, what I mean... is that Drive should do it and return bytes. And then ABCI can have a method to deserialize the bytes.